site/docs/guidelines/header.mdx

```diff
@@ -20,7 +20,7 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 - Anatomy:
     - **Navigation bar:** Contains product switcher (between Engagement and Performance) or Culture Amp logo for customers with only 1 product.
     - **Title Block:** Contains the page title, breadcrumb, primary action, default action, secondary actions, and secondary navigation.
-    - **Skirt:** Draws the user's eye into the content area. Contains the [Filter Drawer](/components/filter-drawer/), viewed or sorted by functionality ([Select](/components/select/)), and page [Search Field](/components/search-field).
+    - **Skirt:** Draws the user's eye into the content area. Viewed or sorted by functionality ([Select](/components/select/)), and page [Search Field](/components/search-field).
```

Comment:
```suggestion
    - **Skirt:** Draws the user's eye into the content area.
```
The rest of the sentence that begins, "Viewed or sorted..." is referring to the Filter Drawer.

Reply:
@sebpearce I updated this.